manifests/n/NelsonNumericalSoftware/Nelson/1.15.0.5475/NelsonNumericalSoftware.Nelson.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.2.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 1.15.0.5475
+PackageLocale: en-US
+Publisher: Nelson numerical software (Allan CORNET)
+PublisherUrl: https://github.com/nelson-lang/nelson/
+PublisherSupportUrl: https://github.com/nelson-lang/nelson/issues
+#PrivacyUrl:
+Author: Allan CORNET
+PackageName: Nelson
+PackageUrl: https://nelson-lang.github.io/nelson-website/
+License: Open Source license LGPLv3
+LicenseUrl: https://raw.githubusercontent.com/nelson-lang/nelson/master/LICENSE
+#Copyright:
+#CopyrightUrl:
+ShortDescription: multi-paradigm programming language and numeric computing environment.
+Description: Nelson is an array programming language providing a powerful open computing environment for engineering and scientific applications using modern C/C++ libraries (Boost, Eigen, …) and others state of art numerical libraries.It has sophisticated data structures (including cell, struct, linear systems, …), an interpreter and a high level programming language. Nelson has been developped to be an open/modular system where an user can define these own data types and operations on these data types by using overload.
+Moniker: nelson
+Tags:
+  - matrix
+  - data
+  - plot
+  - interpreter
+  - language
+  - programming
+  - scientific
+ManifestType: locale
```

A multi file manifest must contain at least version, installer and defaultLocale manifest.
```suggestion
ManifestType: defaultLocale
```